APIClientRead.cs

```diff
@@ -0,0 +1,63 @@
+﻿using API.Automation.Auth;
+using API.Automation.Models.Request;
+using RestSharp;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace API.Automation
+{
+    public class APIClientRead : IAPIClientRead, IDisposable
+    {
+        readonly RestClient client;
+        const string BASE_URL = "http://localhost:49000";
```

Comment:
You need to store this BASE_URL in one place, but not in both classes at the same time. Consider additional options to store this url - base class, config file etc.